### PR TITLE
Mangalib presence

### PR DESCRIPTION
**Pull request by Mirazex**
This presence is for website Mangalib. A popular site for reading manga in my region

DiscordCanary_9PIu7aFgiE
DiscordCanary_76WFPZinx0
DiscordCanary_bXktk5N2oO
DiscordCanary_LFeIGhJ93x
DiscordCanary_vwClYLJRqK

UPDATED:

ki4r140A0R

### CODE REVIEW

File: Mangalib/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "Mirax",
+    "id": "223355525826347018"
+  },
+  "contributors": [{
+    "name": "Mirazex",
+    "id": "223355525826347018"
+  }],
```
Comment (Bas950) [inline Mangalib/dist/metadata.json]: Remove

File: Mangalib/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "service": "Mangalib",
+  "description": {
+    "ru": "Сайт для прочтения манги на русском языке."
+  },
```
Comment (Bas950) [inline Mangalib/dist/metadata.json]: you need an english description aswell

File: Mangalib/dist/presence.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+})
+
+let startDate = Math.floor(Date.now() / 1000)
+
+presence.on("UpdateData", () => __awaiter(this, void 0, void 0, function* () {
+    let route = document.location.href.split("/").slice(2);
+    let presenceData = {
+        largeImageKey: "mangalib_large",
+        smallImageKey: "developer_image",
+        smallImageText: "by Mirax"
```
Comment (Bas950) [inline Mangalib/dist/presence.js]: You are not allowed to have this

File: Mangalib/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "Mirax",
+    "id": "223355525826347018"
+  },
+  "contributors": [{
```
Comment (Alanexei) [inline Mangalib/dist/metadata.json]: Remove yourself as a contributor

File: Mangalib/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "Mirazex",
+    "id": "223355525826347018"
+  }],
+  "service": "Mangalib",
+  "description": {
```
Comment (Alanexei) [inline Mangalib/dist/metadata.json]: Try and add an english description

**Comment by Alanexei** [comment]
Screenshots ^^

**Comment by Bas950** [comment]
Maybe use the reading.png as small image key? https://github.com/PreMiD/Presences/tree/master/.resources

**Comment by Mirazex** [comment]
> Maybe use the reading.png as small image key? https://github.com/PreMiD/Presences/tree/master/.resources

how i can use from resources? Download and load to application from discord/developers or what?

**Comment by Alanexei** [comment]
You take a resource from `.resources` and upload it to the dev application your using for this presence, then you use it as a smallImageKey.

**Comment by Mirazex** [comment]
> You take a resource from `.resources` and upload it to the dev application your using for this presence, then you use it as a smallImageKey.

done

**Comment by Alanexei** [comment]
One more screenshot and we'll be good

**Comment by Mirazex** [comment]
> One more screenshot and we'll be good

gif added

**Pull request merged**